flocker/node/agents/test/test_cinder.py

```diff
@@ -61,3 +67,39 @@ def test_verify_false_ca_path(self):
             'verify_ca_path': '/a/path'
         }
         self.assertEqual(_openstack_verify_from_config(**config), False)
+
+
+@implementer(ICinderVolumeManager)
+class FakeCinderClient(object):
+    def delete(self, volume_id):
+        """
+        A no-op delete
+        """
+
+    def get(self, volume_id):
+        """
+        A no-op get.
+        """
+
+
+class CinderDestroyTests(SynchronousTestCase):
+    def test_timeout(self):
+        """
+        ``CinderBlockDeviceAPI.destroy_volume`` raises
+        ``TimeoutException`` if the volume is not removed in 60s.
```

Comment:
the 1 second thing was because if this tests is going to run periodically, maybe we don't want it to get stuck for 300 seconds. I still need to find out how to do a proper acceptance test. Any ideas are more than welcome!